package.json

```diff
@@ -55,12 +55,11 @@
     "@babel/plugin-syntax-do-expressions": "^7.24.7",
     "@babel/plugin-syntax-function-bind": "^7.24.7",
     "@babel/preset-react": "^7.24.7",
-    "@types/eslint": "=7.2.10",
     "@types/estree": "0.0.52",
     "@types/node": "^4.9.5",
     "@typescript-eslint/parser": "^2.34.0 || ^3.10.1 || ^4 || ^5 || ^6.20 || ^7.14.1 || ^8.4",
     "babel-eslint": "^8 || ^9 || ^10.1.0",
-    "eslint": "^3 || ^4 || ^5 || ^6 || ^7 || ^8 || ^9.7",
+    "eslint": "^3 || ^4 || ^5 || ^6 || ^7 || ^8 || ^9.10",
```

Comment:
unfortunately this is a breaking change, and we must stick to 9.7 as the lowest threshold.

Reply:
There are two `^9.7` in `package.json`. One is in `peerDependencies` and another is in `devDependencies`. I only changed the one in `devDependencies`. Is it going to be a breaking change in this way?

Reply:
It means we won't be able to test on 9.7, 9.8, and 9.9, which means it will break and we won't know about it. Peer and dev dep ranges must always be identical.

Reply:
Okey let me check it back.